ckan/cli/db.py

```diff
@@ -0,0 +1,68 @@
+# encoding: utf-8
+
+import os
+
+import click
+from flask import Flask, current_app
+from werkzeug.serving import run_simple
+
+from ckan.cli import click_config_option
+
+
+@click.group(name=u'db', short_help=u'Database commands')
+@click.help_option(u'-h', u'--help')
+def db():
+    pass
```

Comment:
if you run db without a subcommand it should print the help, not do nothing right?

Reply:
it lists Usage, Options and Commands sections without doing anything else